src/service/video-manager-impl.js

```diff
@@ -281,42 +281,48 @@ export class VideoManager {
   }
 
   /**
-   * Returns the entry in the video manager corresponding to the video
-   * provided
-   *
-   * @param {!../video-interface.VideoInterface} video
+   * Returns the entry in the video manager corresponding to the video or
+   * element provided
+   * @param {!../video-interface.VideoOrBaseElementDef|!Element} videoOrElement
    * @return {VideoEntry} entry
-   * @private
    */
-  getEntryForVideo_(video) {
+  getEntry_(videoOrElement) {
     for (let i = 0; i < this.entries_.length; i++) {
-      if (this.entries_[i].video === video) {
-        return this.entries_[i];
+      const entry = this.entries_[i];
+      if (
+        entry.video === videoOrElement ||
+        entry.video.element === videoOrElement
+      ) {
+        return entry;
       }
     }
-    dev().error(TAG, 'video is not registered to this video manager');
```

Comment:
Erroring and then returning null was weird before, and it's weird now.

This should just assert instead, but I'll explore refactoring this method later anyway. Unsure why the previous conditional check has not been failing before.